server/engine/src/main/java/org/eclipse/lsp/cobol/cli/command/ListCopybooks.java

```diff
@@ -93,7 +93,11 @@ static class Args {
   @Override
   public Integer call() throws Exception {
     if (args.workspaceConfig != null) {
-      parent.initProcessorGroupsReader(args.workspaceConfig.workspace);
+      try {
+        parent.initProcessorGroupsReader(args.workspaceConfig.workspace);
+      } catch (Exception e) {
+        return 2;
```

Comment:
Should be a defined constant with a comprehensive name

Reply:
I can, but was only following the other cases throughout the Analysis code where they use a number.

Reply:
I understand that it is not your code, but can you define a constant and substitute this 2 number with that constant?